allure-report-face/src/util/attachmentType.js

```diff
@@ -24,6 +24,8 @@ export default function typeByMime(type) {
             return 'html';
         case 'text/csv':
             return 'csv';
+        case 'text/tab-separated-values':
```

Comment:
in current realization it songs wrong, because we use different d3js methods for parse cvs and tsv